lib/rules/indentation.js

```diff
@@ -2,22 +2,63 @@
 
 var helpers = require('../helpers');
 
+/**
+ * Check if a node is a newline character or not
+ *
+ * @param {Object} node - The node to test
+ * @returns {boolean} Whether the node is a newline or not
+ */
+var isNewLine = function (node) {
+  // using type === instead of is just in case node happens to be a string
```

Comment:
If you pass it a node though that is a string then `node` will be truthy but instead of `node.type` being undefined it would throw a type error that `node.is is not a function` I think it's just safer overall when dealing with these nodes as we're blindly passing the next node in the sequence to it rather than check if it's a space first, which is what this function is made to do.

So yeah, I'd rather keep these this way.